just/clippy.just

```diff
@@ -0,0 +1,52 @@
+# Check all code `(default)`
+all: native wasm
+
+# Check all native code
+native:
+    # Checking all native code with Clippy
+    __GEAR_WASM_BUILDER_NO_BUILD=1 SKIP_WASM_BUILD=1 \
+      cargo clippy \
+        --locked --workspace --all-targets --all-features \
+        -- --no-deps -D warnings
+
+# Check all WASM code
+wasm:
+    # Checking all WASM code with Clippy
+    cargo metadata --no-deps --format-version=1 \
```

Comment:
Consider to use shebang scripts and paste lines directly from `./scripts/src/clippy.sh`